File: app/src/main/java/com/danilomendes/figurines/util/L.java
package com.danilomendes.figurines.util;

import android.util.Log;

/**
 * Created by danilo on 22-10-2017.
 */
public class L {
    private static final String TAG = "Figurines";

    public static void log(String message) {
        Log.d(TAG, ":" + Thread.currentThread().getName() + " " + message);
    }

    public static void log(String message, Throwable throwable) {
        Log.d(TAG, ":" + Thread.currentThread().getName() + " " + message, throwable);
    }
}

File: app/src/main/java/com/danilomendes/figurines/util/L.kt
package com.danilomendes.figurines.util

import android.util.Log

/**
 * Created by danilo on 22-10-2017.
 */
object L {
    private val TAG = "Figurines"

    fun log(message: String) {
        Log.d(TAG, ":" + Thread.currentThread().name + " " + message)
    }

    fun log(message: String, throwable: Throwable) {
        Log.d(TAG, ":" + Thread.currentThread().name + " " + message, throwable)
    }
}
